### PR TITLE
Added current maintainers.

### PR DESCRIPTION
**Pull request by hsbt**
Followed up https://github.com/rubygems/rubygems/pull/2417

Added the current maintainers.

### CODE REVIEW

**Comment by segiddins** [comment]
I guess we’re all named Evan Phoenix ;)

**Comment by hsbt** [comment]
@segiddins oops, Fixed. Thanks

File: contributing.md [hunk 1]
```diff
@@ -225,6 +248,15 @@ many machines systematically and repeatably.
   <a href="https://github.com/carllerche">
     <img src="https://secure.gravatar.com/avatar/da5274b27cc6c0f505495bf5d504575d?s=32" title="Carl Lerche">
   </a>
+  <a href="https://github.com/indirect">
```
Comment (segiddins) [inline contributing.md]: this is including @indirect twice in this block of links?

**Comment by segiddins** [comment]
@bundlerbot r+

**Pull request merged**